tokens/src/styleDictionary.ts

```diff
@@ -43,6 +43,15 @@ export const getStyleDictionaryConfig = (
           format: 'javascript/module',
           filter: 'isNotFigma',
         },
+        {
+          destination: 'js/tokens.module.reduced.meta.js',
```

Comment:
The output is short but not minified so the min suffix shouldn't be on the file (I think so)

Reply:
I meant that we might also want to generate a minified version to keep consistency with the rest of files, although given the length of the file it will be probably useless. Up to you guys!